MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Dark.xaml

```diff
@@ -20,6 +20,10 @@
     <SolidColorBrush x:Key="MaterialDesignDivider" Color="#1FFFFFFF" po:Freeze="True" /><!-- 12% -->
     <SolidColorBrush x:Key="MaterialDesignSelection" Color="#757575" po:Freeze="True" />
 
+    <SolidColorBrush x:Key="MaterialDesignToolButtonForeground" Color="#616161" po:Freeze="true" />
+    <SolidColorBrush x:Key="MaterialDesignFlatDefaultToggleButtonForeground" Color="#616161" po:Freeze="true" />
+    <SolidColorBrush x:Key="MaterialDesignFlatDefaultToggleButtonBackground" Color="#e0e0e0" po:Freeze="true" />
```

Comment:
I think we want to make the name of these brushes a bit more generic rather than specific to a single style.
So rather than `MaterialDesignFlatDefaultToggleButtonForeground` do `MaterialDesignToolForeground` and rather than  `MaterialDesignFlatDefaultToggleButtonBackground` do `MaterialDesignToolBackground` 

You will also need to update the usages accordingly.

Reply:
Changed them to default because the default is not that visible in a dark theme. As a suggestion can we change the default to the colors on this commit.